packages/terra-list/src/List.jsx

```diff
@@ -13,41 +18,61 @@ const propTypes = {
    */
   children: PropTypes.node,
   /**
-   * Whether or not the child list items should have a border color applied.
+   * Whether or not the list's child items should have a border color applied.
    */
   isDivided: PropTypes.bool,
   /**
    * Function callback for the ref of the ul.
    */
   refCallback: PropTypes.func,
+  /**
+   * Accessibility role of the list, defaults to 'listbox'. If creating a read-only list, pass 'none', and no role will be added.
+   */
+  role: PropTypes.string,
 };
 
 const defaultProps = {
   children: [],
   isDivided: false,
+  role: 'listbox',
```

Comment:
I'm on the fence defaulting this role to `listbox`. I think I understand why it is this way, since isSelectable is now set on on items, there's no clear way to change this role when an item is set to `isSelectabale` since `List` is a parent of `Item`. Defaulting to `listbox` means when people make selectable lists, which is probably the most common use of lists, they get the right role out of the box. On the flip side, people will need to remember to set role to `none` if they aren't doing selectable lists, which feels like it should be the default with using `List` if you aren't aware of the concern were trying to handle with isSelectable now being set on `Item`.

Another way to resolve this would be to introduce `<SelectableList />` which is a copy of `List` with the only difference being the role set. 

`List` would default to no role, `SelectableList` would default to `role="listbox"`

Reply:
After offline discussion, we'll remove the default listbox role, and instead assign it the default of 'none'. Relying on documentation and accessibility failures to encourage consumers to correctly implement the 'listbox' role.